client/ayon_core/tools/workfiles/widgets/window.py

```diff
@@ -113,6 +113,7 @@ def __init__(self, controller=None, parent=None):
 
         main_layout = QtWidgets.QHBoxLayout(self)
         main_layout.addWidget(pages_widget, 1)
+        main_layout.setContentsMargins(0, 0, 0, 0)
```

Comment:
Maybe we should set `pages_widget.layout().setContentsMargins(0, 0, 0, 0)` rather than on main layout.

Reply:
I had my doubts about picking either of the two - I'm fine with swapping them, the result should be quite similar.
Do you want me to move it there?

Reply:
If this `pages_widget.layout().setContentsMargins(0, 0, 0, 0)` work, I would use it, but maybe as
```python
pages_layout = pages_widget.layout()
pages_layout.setContentsMargins(0, 0, 0, 0)
```

Reply:
So I gave this a go. Interestingly `pages_widget.setContentsMargins(0, 0, 0, 0)` also seemed to raise no errors... however, both your example and `pages_widget.setContentsMargins(0, 0, 0, 0)` had no visible effect on the margins of the dialog and hence did not give the intended result for me.

As such, that leaves the current options - or I might be doing something wrong.

Reply:
I think I know why, but that's for future. Approving...